plugins/include/string.inc

```diff
@@ -168,6 +173,23 @@ native format(output[], len, const format[], any:...);
 native formatex(output[], len, const format[], any:...);
 
 /**
+ * Formats and returns a string according to the AMX Mod X format rules
+ * (see documentation).
+ *
+ * @note Example: menu_additem(menu, fmt("My first %s", "item")).
+ * @note This should be used only for simple and unique inline formatting
+ *       like above example.  Do not use this function to store formated
+ *       string into a variable, use other formating functions.
```

Comment:
If we want to comment on the overhead involved something like this might work?

@note This should only be used for simple inline formatting like in the above example. Avoid using this function to store strings into variables as an additional copying step is required.